components/eamxx/tests/multi-process/dynamics_physics/homme_shoc_cld_p3_rrtmgp_pg2/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ CreateDynamicsLib("theta-l_kokkos"  4   72   10)
 # Create the test
 CreateADUnitTest(${TEST_BASE_NAME}
   LABELS dynamics tms shoc cld p3 rrtmgp physics pg2
-  LIBS cld_fraction nudging tms shoc p3 scream_rrtmgp ${dynLibName} diagnostics
+  LIBS cld_fraction tms shoc p3 scream_rrtmgp ${dynLibName} diagnostics
```

Comment:
What is the reason for this change?

Reply:
The test does not use nudging. So if you want to build just this test (while debugging), you would have also had to build nudging, for no reason.